felix/fv/bpf_test.go

```diff
@@ -5509,16 +5509,20 @@ func bpfWaitForPolicy(felix *infrastructure.Felix, iface, hook, policy string) s
 }
 
 func bpfDumpRoutes(felix *infrastructure.Felix) string {
-	var (
-		out string
-		err error
-	)
-
 	if felix.TopologyOptions.EnableIPv6 {
-		out, err = felix.ExecOutput("calico-bpf", "-6", "routes", "dump")
-	} else {
-		out, err = felix.ExecOutput("calico-bpf", "routes", "dump")
+		return bpfDumpRoutesV6(felix)
```

Comment:
I'd keep this the way it is . It only burries the error deeper. Perhaps we should use Expect with offset in the original code to know where we are failing in the test.